doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst

```diff
@@ -243,44 +243,44 @@ Results
 
 .. code-block:: text
 
- particle_ID    T_x       T_y       T_z      f_x       f_y       f_z      f_xv     f_yv     f_zv     f_xp     f_yp      f_zp    
- 	0   -0.000008  0.000019 -0.000022 0.412183  0.000019  0.000038 0.143783 0.000019 0.000038 0.268400 -0.000000  0.000000 
- 	0    0.000001  0.000000 -0.000003 0.415760  0.000001  0.000000 0.162431 0.000001 0.000001 0.253330  0.000000 -0.000000 
- 	0   -0.000000 -0.000000 -0.000000 0.424786 -0.000000 -0.000003 0.176203 0.000001 0.000000 0.248584 -0.000001 -0.000003 
+  particle_ID    T_x      T_y       T_z      f_x       f_y       f_z      f_xv     f_yv      f_zv     f_xp      f_yp      f_zp
+          0   -0.000010 0.000019 -0.000041 0.412175  0.000036  0.000026 0.143775 0.000036  0.000026 0.268400  0.000000  0.000000
+          0   -0.000000 0.000001 -0.000007 0.415760  0.000006 -0.000001 0.162430 0.000006 -0.000001 0.253330 -0.000000 -0.000000
+          0   -0.000000 0.000000 -0.000001 0.424786 -0.000002 -0.000002 0.176205 0.000001 -0.000000 0.248581 -0.000002 -0.000001
 
           
 .. note:: 
-	Because this analysis concerns non-Newtonian flow, there is no known solution for the drag coefficient. For a Newtonian flow at :math:`Re = 50`, the drag force would be :math:`0.6165`. Therefore, the drag force was decreased using a shear-thinning fluid.
+    Because this analysis concerns non-Newtonian flow, there is no known solution for the drag coefficient. For a Newtonian flow at :math:`Re = 50`, the drag force would be :math:`0.6165`. Therefore, the drag force was decreased using a shear-thinning fluid.
 
 -----------------------------
 Possibilities for Extension
 -----------------------------	
 
 * **High-order methods** : Lethe supports higher order interpolation. This can yield much better results with an equal number of degrees of freedom than traditional second-order (Q1-Q1) methods, especially at higher Reynolds numbers.
 * **Reynolds number** : By changing the inlet velocity, it can be interesting to see the impact of the shear-thinning behavior on the effective drag force.
-* **Non-Newtonian parameters** : It can also be interesting to change the Carreau model parameters, i.e. changing the slope to appreciate the change in behavior.
+* **Non-Newtonian parameters** : It can also be interesting to change the Carreau model parameters, i.e. changing the slope to appreciate the behavior change.
 
 .. note::
-	It is not possible to use the Carreau model for a shear-thickening flow. You should change the model to power-law, and use the decribed parameters in :doc:`../../../parameters/cfd/physical_properties`
+    It is not possible to use the Carreau model for a shear-thickening flow. You should change the model to power-law, and use the decribed parameters in :doc:`../../../parameters/cfd/physical_properties`
```

Comment:
Is this necessary? it's a limitation of the model itself, not the numerics. It really is not made for that. Even Pierre Carreau says it, so it has to be true.

Reply:
Not necessary, but I think Pierre Carreau himself would appreciate having us remind people not to do what he says is not meant to be done :upside_down_face:

Reply:
it's chill to keep it like this.